src/pivotal-ui/components/pui-variables.scss

```diff
@@ -321,6 +321,7 @@ $font-family-sans-serif:  'Open Sans', 'OpenSans', sans-serif !default;
 $font-family-serif:       Georgia, "Times New Roman", Times, serif !default;
 $font-family-monospace:   Menlo, Consolas, "Courier New", monospace !default;
 $font-family-base:        $font-family-sans-serif !default;
+$font-cap-height-ratio:   0.73;
```

Comment:
Actually, on third thought, I'm going to advocate for this as matching the status quo: font-awesome only aligns to other fonts by happenstance -- it lands on the baseline and picks a reasonable height to match a line of text. Generally type will have a similar ratio of height to cap-height, plus or minus a few percent here.

I think I should move this variable into the icon component, however, as it's not generally applicable, useful, or useful to tune in other components.